src/containers/navigation-icons/AccountIcon.tsx

```diff
@@ -37,6 +37,20 @@ const AccountIcon: FC<AccountIconProps> = ({ openMenu }) => {
     defaultResponse: defaultResponses.object as UserResponse
   })
 
+  const { photo: statePhoto } = useAppSelector((state) => state.editProfile)
+
+  const getAvatarSrc = useCallback(() => {
+    if (statePhoto?.src) {
+      return statePhoto.src
+    }
+
+    if (photo) {
+      return createUrlPath(import.meta.env.VITE_APP_IMG_USER_URL || '', photo)
+    }
+  }, [photo, statePhoto])
+
+  const avatarSrc = getAvatarSrc()
```

Comment:
The final result of this code is just a variable, that's why it is better to use `useMemo()` instead of `useCallback()`